clouway-pcache-client-gae/src/main/java/com/clouway/api/pcache/extensions/gae/GAECacheManager.java

```diff
@@ -72,7 +75,21 @@ public Object get(String key) {
     return result;
   }
 
-  /**
+    @Override
+    public List<Object> getAll(List<String> keys) {
```

Comment:
You can return: <code>Map<String, Object></code> as the client would be able to check with O(1) speed which keys are returned and which not. 

The keys() and values() of Map are returned as set and sets are having easiest way for finding of union and diff. 

https://stackoverflow.com/questions/18644579/getting-the-difference-between-two-sets